process-create-data-repo-request

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python3
+import re
+from datetime import datetime
+from optparse import OptionParser
+from os.path import dirname, abspath
+from os.path import expanduser
+from socket import setdefaulttimeout
+from typing import Optional, Any
+
+import github
+
+from categories_map import CMSSW_CATEGORIES
+
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+from github import Github
+
+from cms_static import (
+    GH_CMSSW_ORGANIZATION,
+    GH_CMSSW_REPO,
+    CREATE_REPO,
+)
+from github_utils import api_rate_limits, set_comment_emoji
+from os import environ
+
+setdefaulttimeout(120)
+
+JENKINS_PREFIX = "jenkins"
+try:
+    JENKINS_PREFIX = environ["JENKINS_URL"].strip("/").split("/")[-1]
+except (KeyError, IndexError):
+    JENKINS_PREFIX = "jenkins"
+
+try:
+    CMS_BOT_DIR = dirname(abspath(__file__))
+except NameError as e:
+    from sys import argv
+
+    CMS_BOT_DIR = dirname(abspath(argv[0]))
+#
+# Processes a github issue to check if it is requesting the creation of a new data repo
+# If the issue is not requesting new repo, it ignores it.
+
+# -------------------------------------------------------------------------------
+# Global Variables
+# --------------------------------------------------------------------------------
+
+APPROVAL_COMMENT = "^[+]1$"
+opts: Optional[Any] = None
+labels = []
+
+INVALID_REQUEST_MSG = "No category found for requested package {package}"
+EXISTS_MSG = "Requested repository {repo} already exists"
+ACK_MSG = (
+    "Request received. I will create the requested repository after one of the following approve "
+    'the issue: {approvers_list}. You can do this by writing "+1" in a '
+    "comment."
+)
+COMPLETE_MSG = "Repository created: {url}"
+
+# -------------------------------------------------------------------------------
+# Statuses
+# --------------------------------------------------------------------------------
+# This is to determine the status of the issue after reading the labels
+
+# The issue has just been created
+NEW_ISSUE = "NEW_ISSUE"
```

Comment:
shouldn't this be `pending-assignment` ?

Reply:
Yes, I was thinking a bit backwards.